Catppuccin VS Themes/Catppuccin Latte.vstheme

```diff
@@ -10848,5 +10860,1921 @@
     </Category>
+    <Category Name="Azure Resource Manager Tools" GUID="{75a05685-00a8-4ded-bae5-e7a50bfa929a}">
+      <Color Name="HTML Server-Side Script">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="Current Statement">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="Current Statement (new context)">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="Call Return">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="Call Return (historical mode)">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="Current Statement (historical mode)">
+        <Foreground Type="CT_INVALID" Source="FF000000" />
+      </Color>
+      <Color Name="ReSharper Current Line Highlight">
+        <Background Type="CT_INVALID" Source="FF000000" />
+        <Foreground Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="vsvim_blockcaret">
+        <Background Type="CT_RAW" Source="FFDC8A78" />
+        <Foreground Type="CT_RAW" Source="FFDCE0E8" />
+      </Color>
+      <Color Name="vsvim_primarycaret">
+        <Background Type="CT_RAW" Source="FFDC8A78" />
+        <Foreground Type="CT_RAW" Source="FFDCE0E8" />
+      </Color>
+      <Color Name="xml doc comment - name">
+        <Foreground Type="CT_RAW" Source="FF8839EF" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="HTML Element Name">
+        <Foreground Type="CT_RAW" Source="FF8839EF" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="CSS Selector">
+        <Foreground Type="CT_RAW" Source="FF8839EF" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="brace pair level three">
+        <Foreground Type="CT_RAW" Source="FFD20F39" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="keyword - control">
+        <Foreground Type="CT_RAW" Source="FFD20F39" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="Preprocessor Keyword">
+        <Foreground Type="CT_RAW" Source="FFD20F39" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="MarkerFormatDefinition/SelectedTextInScrollBar">
+        <Foreground Type="CT_RAW" Source="FFD20F39" />
+      </Color>
+      <Color Name="parameter name">
+        <Foreground Type="CT_RAW" Source="FFE64553" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="delegate name">
+        <Foreground Type="CT_RAW" Source="FFFE640B" />
+        <Background Type="CT_INVALID" Source="00000000" />
+      </Color>
+      <Color Name="enum name">
+        <Foreground Type="CT_RAW" Source="FFFE640B" />
```

Comment:
I believe foreground nodes prior to background nodes might be causing build issues. Please see my comment about this.